app/models/thredded/messageboard.rb

```diff
@@ -50,11 +56,35 @@ class Messageboard < ActiveRecord::Base
 
     has_many :post_moderation_records, inverse_of: :messageboard, dependent: :delete_all
 
-    default_scope { where(closed: false).order(topics_count: :desc) }
-
+    default_scope { where(closed: false) }
+    # rubocop:disable Style/Lambda
     scope :top_level_messageboards, -> { where(group: nil) }
     scope :by_messageboard_group, ->(group) { where(group: group.id) }
-
+    scope :ordered, ->(order = Thredded.messageboards_order) {
+      case order
+      when :position
+        ordered_by_position
+      when :created_at_asc
```

Comment:
It still orders by position in the `ordered_by_created_at_asc` though. And position might not be the same as `created_at`.